db/seeds.rb

```diff
@@ -1,3 +1,20 @@
+# Create a user with application and events
+user = User.create( email: Fake.email,
+                    password: "helloworld",
+                    confirmed_at: Time.zone.now
+)
+app_name = Fake.application_name
+application = Application.create( name: app_name,
```

Comment:
Favor using the bang-methods for creating data in seeds.rb (`Application.create!(...)` for example). The bang methods will mean that if test data fails to be created an error will be raised so developers are aware of the problem immediately.